sigs.yaml

```diff
@@ -30,20 +30,22 @@ sigs:
     frequency: weekly
     url: meet.google.com/ngp-npcx-nws
     archive_url: https://docs.google.com/document/d/19_xPxfsazD6rJfe1aHNjVC9_bKpOfnepsifCZ4GBw8o/edit
+  - description: OS-Climate Project Meeting
+    day: Thursday
+    time: "11:00"
+    tz: ET (East Time)
+    frequency: weekly
+    url: meet.google.com/kdy-sqyf-rud
+    archive_url: https://docs.google.com/document/d/1PZTRTrU68LZXUy9GgKCp38KpVyG4lrN5Cw8Zv9pGmjE/edit
   contact:
     liaison:
       github: michaelclifford
       name: Michael Clifford
   subprojects:
-  - name: AIOps
-    owners:
-    - https://raw.githubusercontent.com/open-services-group/community/main/sig-data-science/OWNERS
-  - name: MLOps
-    owners:
-    - https://raw.githubusercontent.com/open-services-group/community/main/sig-data-science/OWNERS
-  - name: Monitoring
+  - name: OS-Climate
+    description: '[Described below](#os-climate)'
     owners:
-    - https://raw.githubusercontent.com/open-services-group/community/main/sig-data-science/OWNERS
+    - https://github.com/os-climate/aicoe-osc-demo/blob/master/OWNERS
```

Comment:
@erikerlandson  are there other repos we should track here as being "owned" by this subproject?

Reply:
a few more:
https://github.com/os-climate/osc-trino-acl-dsl/blob/main/OWNERS
https://github.com/os-climate/osc-ingest-tools/blob/main/OWNERS
https://github.com/os-climate/data-platform-demo

Reply:
oops should be:
https://github.com/os-climate/data-platform-demo/blob/master/OWNERS

Reply:
Thanks!

Reply:
One last question - are these supposed to be to raw-files, like:
https://raw.githubusercontent.com/os-climate/aicoe-osc-demo/master/OWNERS

Reply:
good catch, yes they need to be the raw files. Thanks I'll make the change